src/moj/components/password-reveal/password-reveal.spec.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @jest-environment jsdom
+ */
+const { getByDisplayValue, getByText } = require("@testing-library/dom");
+const { axe, toHaveNoViolations } = require('jest-axe')
+expect.extend(toHaveNoViolations)
+
+
+require('../../../../jest.setup.js')
```

Comment:
I think you should be able to use [setupFiles](https://jestjs.io/docs/configuration#setupfiles-array) in a Jest config if you want to avoid needing a manual import